src/storage/mod.rs

```diff
@@ -1753,4 +1786,99 @@ mod tests {
         );
         storage.stop().unwrap();
     }
+
+    #[test]
+    fn test_raw_delete_range() {
+        let read_pool = ReadPool::new("readpool", &readpool::Config::default_for_test(), || {
+            read_pool_context_factory
+        });
+        let config = Config::default();
+        let mut storage = Storage::new(&config, read_pool).unwrap();
+        storage.start(&config).unwrap();
+        let (tx, rx) = channel();
+
+        // Write some key-value pairs to the db
+        for kv in &[
```

Comment:
Assign these samples to a variable so that you don't need to list them below again.